src/index.ts

```diff
@@ -8,6 +8,7 @@ import {
   PreviewMode,
   MosaickingOrder,
   Interpolator,
+  DEMInstanceType,
```

Comment:
Maybe `DEMType` would be a better variable name? At least I associate "instance" with WMS... Fftr.

Reply:
I see now that this is actually in line with the naming on SH service, resolving.